Model/src/GamePackage/Player.java

```diff
@@ -1,4 +1,265 @@
+	 * 
+	 * @param propertiesOwned Sets the ArrayList propertiesOwned.
+	 *****************************************************************/
+	public void setPropertiesOwned(ArrayList<BoardSquare> 
+	propertiesOwned) {
+		this.propertiesOwned = propertiesOwned;
+	}
+
+	/******************************************************************
+	 * Getter method for cardsHeld ArrayList.
+	 * 
+	 * @return cardsHeld Returns the ArrayList cardsHeld.
+	 *****************************************************************/
+	public ArrayList<Card> getCardsHeld() {
+		return cardsHeld;
+	}
+
+	/******************************************************************
+	 * Setter method for cardsHeld ArrayList.
+	 * 
+	 * @param cardsHeld Sets the ArrayList cardsHeld.
+	 *****************************************************************/
+	public void setCardsHeld(ArrayList<Card> cardsHeld) {
+		this.cardsHeld = cardsHeld;
+	}
+	
+	/******************************************************************
+	 * Checks to make sure player has sufficient funds before
+	 * subtracting amount from wallet. If the player has sufficient
+	 * funds the method subtracts amount from wallet and returns the
+	 * amount paid. If player doesn't have sufficient funds he method
+	 * subtracts what is able to be paid from wallet and returns the
+	 * amount that has been paid.
+	 * 
+	 * @param amount The amount of money requested from player.
+	 * @return amountPaid The amount of money that was paid by player.
+	 *****************************************************************/
+	public int pay(int amount) {
+		int amountPaid = 0;
+		
+		if(wallet >= amount) {
+			wallet -= amount;
+			amountPaid = amount;
+		}
+		else {
+			amountPaid = amount - wallet;
+			wallet -= amountPaid;
+		}
+		
+		return amountPaid;
+	}
+	
+	/******************************************************************
+	 * Removes the property from the player's ArrayList of properties
+	 * owned.
+	 * 
+	 * @param property The property to be removed from owned properties
+	 * list.
+	 *****************************************************************/
+	public void sell(BoardSquare property) {
+		propertiesOwned.remove(property);
+	}
+	
+	/******************************************************************
+	 * Mortgages the player's property and adds the mortgage value to
+	 * their wallet.
+	 * 
+	 * @param property The property to be mortgaged.
+	 *****************************************************************/
+	public void mortgage(BoardSquare property) {
+		wallet += property.getMortgageValue();
+		property.setIsMortgaged(true);
+	}
+	
+	/******************************************************************
+	 * Adds the property to the player's ArrayList of properties
+	 * owned.
+	 * 
+	 * @param property The property to be added to owned properties
+	 * list.
+	 *****************************************************************/
+	public void giveProperty(BoardSquare property) {
+		propertiesOwned.add(property);
+	}
+	
```

Comment:
The idea was for giveProperty() to be removing a property from the player. As if the player is giving it away. Then there also needs to be a receieveProperty() which adds it.